Source/Private/Helpers/String+Truncation.swift

```diff
@@ -32,11 +32,15 @@ extension String {
                         attributes: [NSAttributedString.Key: Any],
                         trailingTextAttributes: [NSAttributedString.Key: Any]) -> NSAttributedString {
 
-        if !willFit(to: size, attributes: attributes) {
+        if !willFit(to: size,
+                    attributes: attributes,
```

Comment:
Oh, I think I found another bug. When calculating the size, we should also pass the `ellipseString` and `trailingText`.

```suggestion
                    ellipsesString: ellipsesString,
                    trailingText: trailingText,
                    attributes: attributes,
```

Reply:
Fine! Difficult to notice, because the parameter has a default value 🙁

Reply:
Revised again, and came to the conclusion: there is no need to pass the ellipseString, because it is added only if it does not fit.
Reverted commit.